.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,37 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+    - 'master'
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
+    - name: Set up node
+      uses: actions/setup-node@v2
+      with:
+        node-version: '6'
```

Comment:
How badly does it break if we try using a more recent, supported NodeJS version?